extensions/wikia/Email/templates/avatarLayout.mustache

```diff
@@ -1,6 +1,6 @@
 <div class="greeting">
 	<p>{{salutation}}</p>
-	<p>{{{summary}}}</p>
+	<p><strong>{{{summary}}}</strong></p>
```

Comment:
Could we update this to use CSS? Generally you use \<strong\> tags when when you're indicating something more important, but I think this is more of a stylistic decision (The example I've read is "`<p><strong>Do not touch.</strong> Contains <strong>hazardous</strong> materials.</p>`).

Also, I extracted this greeting div out into a partial [here](https://github.com/Wikia/app/pull/7426/files#diff-b467f3bb9d5d158bafee32774e7a8506R1). The WeeklyDigest template doesn't call for the summary being bold, so if we put that CSS inside of avatarLayout.css it'll render properly in both cases.